server/planning/content_profiles/profiles/advanced_search.py

```diff
@@ -157,6 +150,13 @@
                 "search_enabled": True,
                 "filter_enabled": True,
             },
+            "language": {
+                "enabled": True,
+                "index": 1,
+                "group": "language",
```

Comment:
I don't think we need a specific group for 1 input, does this work if changing `enabled` to `True` (with no other changes)?

Reply:
>  does this work if changing enabled to True (with no other changes)?
 yes its works

Reply:
> I don't think we need a specific group for 1 input,

IMO , It is mentioned in the ticket to create a new language group.